packages/webapp/components/FooterNavBar.tsx

```diff
@@ -37,9 +65,51 @@ export default function FooterNavBar({
   post,
 }: FooterNavBarProps): ReactElement {
   const router = useRouter();
-  const { user } = useContext(AuthContext);
+  const { user, squads } = useContext(AuthContext);
   const feedName = getFeedName(router.pathname, { hasUser: !!user });
   const activeNav = useActiveNav(feedName);
+
+  const hasSquads = squads?.length > 0;
+  const squadsUrl = hasSquads
+    ? squadCategoriesPaths['My Squads']
```

Comment:
Got so confused by this string 😂
Not sure why we didn't make it camelCase or something like that.